lib/assessment.js

```diff
@@ -38,8 +54,17 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // If the argument obj is unset, create a new object
 // The function you write should return
 // the object passed in as an argument or the object created
-var addMethod = function addMethod(propertyName, method, obj) {
 
+var addMethod = function addMethod(propertyName, method, obj) {
+  if(typeof arguments[2] === 'undefined') {
+      var newObj = {}
+      newObj[propertyName] = method;
+      return newObj;
+  }
+  else  {
+    obj[propertyName] = method;
+    return obj;
+      }
```

Comment:
This will work, but there's a simpler way. If no value is passed in, `obj` is undefined, so `!obj` would be truthy. Consequently, we could write 
```javascript
if (!obj) { //alternatively, 'unless(obj)'
  obj = {};
}
obj[propertyName] = method;
return obj;
```